packages/sdk/src/swap/v2/types.ts

```diff
@@ -124,10 +111,10 @@ interface Sending extends Receiving {
   refundEgress: EgressFields | undefined;
 }
 
-type SwapState =
+export type SwapStatusResponseV2 =
```

Comment:
as far as i understand, this does not change the resulting type for `SwapStatusResponseV2` at all